docs/release-notes/15-4-0/README.md

```diff
@@ -10,18 +10,102 @@ release_date: 2025-03-06
+
+Starting with OpenProject 15.4, users will be able to choose whether they want to manually schedule their work packages or enable automatic scheduling. This highly requested feature of automatic scheduling, based on predecessor/successor relations, will make managing complex project setups much easier.
+
+> [!NOTE]
+> The new feature will not overwrite existing dates for any existing work packages. However, it can change the scheduling mode. By default, manual scheduling is enabled. Existing work packages with children will be in automatic mode – with dates derived from their children.
+
+![New scheduling modes: Automatic and Manual](manualAutomatic_schedulingModes.png)
+
+In **manual mode**, users can freely select dates and can set timelines based on specific needs, deadlines or external factors. Manually scheduled work packages behave as they did before 15.4.
+
+In the **new automatic mode**, manually setting a start date is not possible. Instead, when a task is scheduled, the date picker automatically determines the start date based on the nearest predecessor. In other words: The work package starts as soon as possible, considering related work packages. However, you can still define a duration, which adjusts the finish date accordingly. This ensures a reliable sequence of automatically scheduled work packages that adapt dynamically to planning changes.
+
+To learn more about why we needed this change and how it might help you manage your projects, please read [this blog article on the new automatic scheduling mode](https://www.openproject.org/blog/new-automatic-scheduling-mode/blog/new-automatic-scheduling-mode/) that we published some weeks ago. See our documentation to get detailed information about how to use the automatic scheduling mode.
+
+### Set agenda item outcomes for meetings
+
+In February 2025, we already published a milestone for our Meetings module: [Recurring meetings](../../release-notes/15-3-0/#schedule-recurring-meetings-to-streamline-your-weekly-biweekly-or-custom-appointments). With OpenProject 15.4, we released another great new feature that will help users better manage their meeting outcomes. They can now note relevant decisions, questions and follow-up actions for each agenda item. This way, all participants as well as users that could not attend the meeting will have a documentation of the relevant meeting outcome(s).
+
+> [!NOTE]
+> In our classic Meetings module, users could write "minutes", which are notes taken during the meeting. Users can still edit the agenda item description to do so, but now they can also use the outcome feature to make it more clear.
+
+The updated Meetings module now allows you to set a meeting status: 
+
+![Screenshot: OpenProject's Meetings module, setting a meeting status](openproject-15-4-meeting-status.png)
+
+Once the status is "in progress", you can add outcomes to every agenda item. If it is a recurring meeting, can also move the agenda item to the next meeting:
```

Comment:
Word missing ', can also move the agenda item.' -> ', you can also move the agenda item'